keyboards/dz60/rules.mk

```
@@ -25,9 +18,7 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 NKRO_ENABLE = yes           # USB Nkey Rollover
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
-MIDI_ENABLE = no            # MIDI support
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
-FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 
-LAYOUTS = 60_ansi 60_ansi_split_bs_rshift 60_hhkb 60_iso 60_abnt2 60_tsangan_hhkb
+LAYOUTS = 60_ansi 60_ansi_arrows 60_ansi_split_bs_rshift 60_hhkb 60_iso 60_abnt2 60_tsangan_hhkb
```

Only the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts?id=layouts-using-a-keymap-with-multiple-keyboards) uses this rule, and `60_ansi_arrows` isn't a valid option there.

```suggestion
LAYOUTS = 60_ansi 60_ansi_split_bs_rshift 60_hhkb 60_iso 60_abnt2 60_tsangan_hhkb
```

---

I've created the required files based on [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts?id=layouts-using-a-keymap-with-multiple-keyboards) for `60_ansi_arrow`